src/map-app/app/map-ui.ts

```diff
@@ -131,10 +132,14 @@ export class MapUI {
       throw new Error(`an attempt to add a filter on an unknown propery name '${propName}'`);
 
     const isMulti = propDef.type === 'multi';
-    if (value === undefined)
+    if (value === undefined) {
       this.stateManager.clearPropFilter(propName);
-    else
+    } else {
       this.stateManager.propFilter(new PropEquality(propName, value, isMulti));
+    }
+
+    if (getViewportWidth() > 1080) // on smaller screens, wait until user clicks Apply Filters
```

Comment:
I know this is a magic number from the stylesheets. But perhaps for self-documentation and to avoid them getting out of sync they should be a named constant?

Or, elsewhere I see the use of a utility function `getViewportWidth()`, which looks a bit like it's supposed to compute this value, perhaps that should be used?

Reply:
Yep I'll add a constant, in both JS and CSS

Reply:
So... I've done it for JS but it's not actually possible to set named constants for css @media queries yet. https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries

There are some hacky workarounds, or we could use some npm libraries to enhance the standard css functionality, but I think that's out of scope for this issue. We only use the constants in 3 places in our CSS stylesheet, so I think it's something we can live with for now.